com.github.Try.OpenGothic.appdata.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.Try.OpenGothic</id>
+  <name>OpenGothic</name>
+  <summary>Open source reimplementation of Gothic 2 Notr</summary>
+  <description>
+    <p>Open source remake of Gothic 2: Night of the raven. Motivation: The original Gothic 1 and Gothic 2 are still great games, but it not easy to make them work on modern systems. The goal of this project is to make a feature complete Gothic client compatible with the game and mods.</p>
+    <p>Core gameplay is done, you can complete the first chapter, as well as all addon content for any guild.</p>
+	<p>Note: OpenGothic does not currently have any sort of GUI launcher, and as such needs to be launched from a terminal. See the GitHub page for instructions on how to do this.</p>
+  </description>
+  <categories>
+    <category>Game</category>
+    <category>AdventureGame</category>
+    <category>RolePlaying</category>
+  </categories>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-realistic">intense</content_attribute>
+    <content_attribute id="violence-bloodshed">mild</content_attribute>
+    <content_attribute id="violence-desecration">intense</content_attribute>
+    <content_attribute id="violence-slavery">moderate</content_attribute>
+    <content_attribute id="drugs-alcohol">moderate</content_attribute>
+    <content_attribute id="drugs-narcotics">moderate</content_attribute>
+    <content_attribute id="drugs-tobacco">moderate</content_attribute>
+    <content_attribute id="sex-nudity">mild</content_attribute>
+    <content_attribute id="sex-themes">intense</content_attribute>
+    <content_attribute id="language-profanity">intense</content_attribute>
+    <content_attribute id="language-humor">moderate</content_attribute>
+    <content_attribute id="language-discrimination">intense</content_attribute>
+  </content_rating>
+  <url type="homepage">https://github.com/Try/OpenGothic</url>
+  <url type="bugtracker">https://github.com/Try/OpenGothic/issues</url>
+  <project_license>MIT</project_license>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Screenshot of the game</caption>
+      <image>https://raw.githubusercontent.com/Try/OpenGothic/master/scr0.png</image>
+    </screenshot>
+  </screenshots>
+  <metadata_license>MIT</metadata_license>
```

Comment:
CC-0 is prefered for the appstream metadata.

Reply:
Might I ask what the reasoning behind that is?

Reply:
https://github.com/flathub/flathub/wiki/App-Requirements#appstream

Also (and that's my personal opinion), MIT is code license. This is (meta)data.

Reply:
I see. While I did write this file, upstream requested I licence it MIT. I can ask how he feels about changing it though.

Reply:
Upstream wants to keep it MIT. As of next release the appdata should come from the OpenGothic repo and installed by CMake instead of being here.

Reply:
Upstream wins on licencing their own code so that's fine although as @hfiguiere says metadata specific licences are preferred for metadata.